### PR TITLE
Completion Suggester: Clarify suggestion indexing

### PR DESCRIPTION
**Pull request by areek**
Clarify how to index multiple suggestion entries per document

### CODE REVIEW

File: docs/reference/search/suggesters/completion-suggest.asciidoc [hunk 1]
```diff
@@ -73,6 +73,11 @@ Mapping supports the following parameters:
 [[indexing]]
 ==== Indexing
 
+You index suggestions like any other field. A suggestion is made of an
+`input` and an optional `weight` attribute. An `input` is the expected
+text to be matched by a suggestion query and the `weight` dgggggetermines how
```
Comment (clintongormley) [inline docs/reference/search/suggesters/completion-suggest.asciidoc]: Leaning on the "g" key?

**Comment by clintongormley** [comment]
Other than the typo, LGTM :)

**Comment by s1monw** [comment]
@areek wanna get it in?

**Comment by areek** [comment]
Thanks for the review. pushed to `feature/completion_suggester_v2` (https://github.com/elastic/elasticsearch/commit/9a9c1e540964c5a887210a685184069c97cbe584)

**Pull request closed without merging**